Kwc/Columns/Trl/Events.php

```diff
@@ -69,4 +80,27 @@ public function onMasterRowDelete(Kwf_Events_Event_Row_Abstract $event)
             }
         }
     }
+
+    public function onMasterOwnRowUpdate(Kwf_Events_Event_Row_Abstract $event)
```

Comment:
könnte da nicht statt der eigenten implementierung einfach throwContentChangedOnOwnMasterModelUpdate (das setting) verwendet werden?

Reply:
Nein leider nicht weil das ganze nicht von trl chained events erbt

Reply:
Kannst du das ändern?

Reply:
nicht in 3.10

Reply:
guter punkt :+1: